<?xml version="1.0"?>
<launch>
	<!-- Turn On ZED 2i Camera (with rviz) -->
	<arg name="svo_file"             default="" /> <!-- <arg name="svo_file" default="path/to/svo/file.svo"> -->
	<arg name="stream"               default="" /> <!-- <arg name="stream" default="<ip_address>:<port>"> -->
	<arg name="camera_model"         default="zed2i" />
	<include file="$(find zed_wrapper)/launch/$(arg camera_model).launch">
        	<arg name="camera_model"        value="$(arg camera_model)" />
        	<arg name="svo_file"            value="$(arg svo_file)" />
        	<arg name="stream"              value="$(arg stream)" />
    	</include>
    	
    	<!-- Config Default Scripts and Nodes, DO NOT EDIT ! -->
	<arg name="model" default="dofbot_with_line.urdf"/>
	<arg name="gui" default="false"/>
	<param name="robot_description" command="$(find xacro)/xacro --inorder $(find dofbot_moveit)/urdf/$(arg model)"/>
	<param name="use_gui" value="$(arg gui)"/>
	
    	
    	<!-- Turn On Merged RViz -->
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
	<node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui"/>
	<!--<node name="calculate_servo_angle" pkg="dofbot_moveit" type="calculate_angle.py"/>-->
	<!--<node name="tf_broadcaster_node" pkg="dofbot_moveit" type="tf_broadcaster.py"/>-->
    	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find dofbot_moveit)/rviz/dofbot_merged_with_zed.rviz" output="screen" />
</launch>
